config/default/traefik.yaml

```diff
@@ -0,0 +1,25 @@
+globalArguments:
+  - "--global.checknewversion"
+logs:
+  general:
+    format: json
+  access:
+    enabled: true
+    format: json
+providers:
+  kubernetesIngress:
+    publishedService:
+      enabled: true
+ingressRoute:
+  dashboard:
+    enabled: disabled
```

Comment:
```suggestion
    enabled: false
```

As per:
* https://github.com/traefik/traefik-helm-chart/blob/master/traefik/values.yaml#L85
* https://github.com/traefik/traefik-helm-chart/blob/4229d69220c98ed2d4f08f1f19d23afd5c7597b6/traefik/templates/dashboard-hook-ingressroute.yaml#L1

Reply:
I brainfarted trying to adapt my own config 😆

Reply:
ah ah don't worry, it was still working, as the value was casted to `false` (and the template's conditional was never triggered).